.github/workflows/linters.yml

```diff
@@ -0,0 +1,48 @@
+name: Linters
```

Comment:
Please add a professional and descriptive  README file as mentioned by the first reviewer. Your repo should look professional before review . Please go through this [lesson](https://microverse.pathwright.com/auth/sign-in/?next=/library/fast-track-curriculum/69047/path/step/66243642/) for more details.